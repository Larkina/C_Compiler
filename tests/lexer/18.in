145.16e+2