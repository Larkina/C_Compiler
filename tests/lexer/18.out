Token{value=14516.0, text=n10, type=INT}